core/common/src/main/java/alluxio/wire/TieredIdentity.java

```diff
@@ -169,6 +170,41 @@ public String getTierName() {
     public String getValue() {
       return mValue;
     }
+
+    /**
+     * @param otherTier wire type locality tier to compare to
+     * @return true if the wire type locality tier is local in comparison to the given tier
+     */
+    public boolean checkLocality(LocalityTier otherTier) {
+
```

Comment:
nit: the formatting convention for this codebase is to not start methods with empty lines

Reply:
I see, I will take that into consideration next time, wondering why checkstyle didn't catch that?

Reply:
unfortunately I don't think checkstyle supports such a check in its standard set of checks. It could be interesting though to add our own custom checker that could check things like this.